src/ray/rpc/grpc_server.cc

```diff
@@ -66,7 +66,12 @@ void GrpcServer::Run() {
   // Create calls for all the server call factories.
   for (auto &entry : server_call_factories_) {
     for (int i = 0; i < num_threads_; i++) {
-      entry->CreateCall();
+      // Create a buffer of 100 calls for each RPC handler.
```

Comment:
Hi @edoakes, we have `n` threads and each thread attached (or poll) a completion queue. And we create `n` ServerCallFactories for each registered service and each attaches to one completion queue. Why here we create `n * 100 * Call` for each ServerCallFactories? Should we just create `100 Call` for each  ServerCallFactories? In the current implementation, for each service:
1 thread will create 100 Calls totally
2 threads will create 400 Calls totally.
3 threads will create 900 Calls totally.
Does this reasonable?

Reply:
Creating 100 calls for each `ServerCallFactory` sounds good.